### PR TITLE
(SIMP-6832) Support zone transfers and dynamic DNS

### PR DESCRIPTION
**Pull request by trevor-vaughan**
- Allow users to force enabling/disabling of the chroot settings
- Allow users to easily set the 'named_write_master_zones' SELinux boolean in
  case they need to support dynamic DNS or zone transfers.

SIMP-6832 #close

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
:hourglass_flowing_sand: https://gitlab.com/simp/pupmod-simp-named/pipelines/76201655

File: manifests/non_chroot.pp [hunk 1]
```diff
@@ -37,15 +33,6 @@
     require => Package['bind']
   }
 
-  file { '/var/named':
```
Comment (lnemsick-simp) [inline manifests/non_chroot.pp]: When you moved this to init.pp, you removed the notify and require.  Why?

**Comment by lnemsick-simp** [comment]
Do you want to create a REFERENCE.md for this project?

File: manifests/non_chroot.pp (hunk 1 above)
Comment (trevor-vaughan) [reply manifests/non_chroot.pp]: @lnemsick-simp They're handled by `autorequires` in the various components and the Package/Dir relationship doesn't matter in terms of order.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp Added the REFERENCE.md

**Pull request merged**